apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -16,12 +16,12 @@
             autocorrect="off"
             {{^editable}}disabled{{/editable}}
             {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
-            aria-describedby="pl-symbolic-input-{{uuid}}-suffix"
-            aria-labelledby="pl-symbolic-input-{{uuid}}-label"
+            aria-describedby="pl-string-input-{{uuid}}-suffix"
```

Comment:
This will only exist if there is a suffix. Should this be conditional?

Reply:
I believe screen readers will just ignore elements that don't exist, so this should be safe. e.g. if you look at this element in the Chrome accessibility inspector prior to this change, it simply doesn't have a label.